src/sentry/incidents/models/alert_rule.py

```diff
@@ -424,6 +424,7 @@ def clear_alert_rule_trigger_cache(cls, instance, **kwargs):
 class AlertRuleThresholdType(Enum):
     ABOVE = 0
     BELOW = 1
+    BOTH = 2
```

Comment:
ehhh?

what does both mean?

Reply:
From the anomaly detection spec: detect anomaly if values are either above or below the threshold

Reply:
Ahhhh yea i see that's right 🤔

Can we add a note for that here?

Reply:
Would naming it `ABOVE_AND_BELOW` be more clear? or is `BOTH` with the note fine?